src/SamplesApp/SamplesApp.Shared/App.xaml.cs

```diff
@@ -476,6 +476,9 @@ static void ConfigureFeatureFlags()
 #endif
 #if __SKIA__
 			Uno.UI.FeatureConfiguration.ToolTip.UseToolTips = true;
+#endif
+#if HAS_UNO
+			Uno.UI.FeatureConfiguration.TextBox.UseOverlayOnSkia = true;
```

Comment:
Isn't it the reverse? The Gtk/WPF ones are overlay-based, not the Skia-based one.

Reply:
Yeah, my bad.

Reply:
Good catch, that would not be dogfooding much 😅!